src/e2e/java/teammates/e2e/util/JMeterElements.java

```diff
@@ -0,0 +1,269 @@
+
+        return testPlan;
+    }
+
+    /**
+     * Returns a Thread Group element.
+     * @param numThreads The number of concurrent threads that will run the test plan and independently of the other threads
+     * @param rampUpPeriod Time (in seconds) to take to "ramp-up" to the full number of threads
+     * @param numLoops The number of times to execute the entire test plan
+     */
+    public static ThreadGroup threadGroup(int numThreads, int rampUpPeriod, int numLoops) {
+        ThreadGroup threadGroup = new SetupThreadGroup();
+
+        threadGroup.setName("Thread Group");
+        threadGroup.setNumThreads(numThreads);
+        threadGroup.setRampUp(rampUpPeriod);
+        threadGroup.setProperty(new StringProperty(ThreadGroup.ON_SAMPLE_ERROR, ThreadGroup.ON_SAMPLE_ERROR_CONTINUE));
+        threadGroup.setSamplerController(loopController(numLoops));
+        threadGroup.setProperty(TestElement.TEST_CLASS, ThreadGroup.class.getName());
+        threadGroup.setProperty(TestElement.GUI_CLASS, ThreadGroupGui.class.getName());
+
+        return threadGroup;
+    }
+
+    /**
+     * Returns a Loop Controller element that configures the number of times its sub-elements are iterated in a test run.
+     * @param loopCount The number of iterations in a test run
+     */
+    public static LoopController loopController(int loopCount) {
+        LoopController loopController = new LoopController();
+
+        loopController.setLoops(loopCount);
+        loopController.setEnabled(true);
+        loopController.setProperty(TestElement.TEST_CLASS, LoopController.class.getName());
+        loopController.setProperty(TestElement.GUI_CLASS, LoopControlPanel.class.getName());
+
+        return loopController;
+    }
+
+    /**
+     * Returns a CSV Data Set Config element that reads lines from a file and splits them into variables.
+     * @param configFilePath Path to the CSV config file
+     */
+    public static CSVDataSet csvDataSet(String configFilePath) {
+        CSVDataSet csvDataSet = new CSVDataSet();
+
+        csvDataSet.setName("CSV Data Config");
+        csvDataSet.setProperty(new StringProperty("filename", configFilePath));
+        csvDataSet.setProperty(new StringProperty("delimiter", "|"));
+        csvDataSet.setProperty(new StringProperty("shareMode", "shareMode.all"));
+        csvDataSet.setProperty("ignoreFirstLine", true);
+        csvDataSet.setProperty("quoted", true);
+        csvDataSet.setProperty("quotedData", true);
+        csvDataSet.setProperty("recycle", false);
+        csvDataSet.setProperty("stopThread", true);
+        csvDataSet.setProperty(TestElement.TEST_CLASS, CSVDataSet.class.getName());
+        csvDataSet.setProperty(TestElement.GUI_CLASS, TestBeanGUI.class.getName());
+
+        return csvDataSet;
+    }
+
+    /**
+     * Returns a HTTP Cookie Manager element.
+     */
+    public static CookieManager cookieManager() {
```

Comment:
Since the method seems to be a getter, you might want to rename it as `getCookieManager` similar to `getJsonData`

Reply:
Yeah, I thought about that but decided against it since all the methods in this utility class return TestElements. I felt it would be better to just name the methods after the elements

Also, this isn't technically a getter, but I understand what you mean :stuck_out_tongue: